homework/index.js

```diff
@@ -30,17 +33,124 @@
     return elem;
   }
 
+  function createDetails(leftColumn, repositories) {
+    const table = createAndAppend('table', leftColumn, { id: 'main-table' });
+    const tBody = createAndAppend('tbody', table);
+    const repoDetails = ['Repository', 'Description', 'Forks', 'Updated'];
```

Comment:
Nice job. I like how you are looping through this rather than just copying and pasting. Shows a very good understanding of programming being about creative laziness. 😄

Reply:
laziness is the mother of  inventions I guess :p